### PR TITLE
✨Model updates

### PR DESCRIPTION
**Pull request by parimalak**
Summary of changes in this issue:
- [x] Add `release_status` field to study
- [x] Add `icd_id`, `mondo_id` ,`uberon_id`fields to diagnosis
- [x] Add `snowmed_id` field to phenotype
- [x] Add `is_harmonized`, `reference_genome` fields to genomic_file 

These changes will be in incorporated in #202:
- [ ] Update `concentration` field to Float for aliquot model
- [ ] Add `uberon_id` field to sample

### CODE REVIEW

**Comment by parimalak** [comment]
Resolves #198

File: tests/aliquot/test_aliquot_models.py [hunk 1]
```diff
@@ -28,23 +28,8 @@ def create_participant_sample_aliquot(self):
...
-        aliquot_data = {
-            'external_id': aliquot_id,
-            'shipment_origin': 'CORIELL',
-            'shipment_destination': 'Broad Institute',
-            'analyte_type': 'DNA',
-            'concentration': 100,
-            'volume': 12.67,
-            'shipment_date': dt
-        }
+        sample_data =self._make_sample(external_id=sample_id)
```
Comment (dankolbman) [inline tests/aliquot/test_aliquot_models.py]: Can we stay consistent with spacing around assignments?

**Review by dankolbman — COMMENTED** [review]
Looks good, though we'll likely refactor the ontological columns soon in a different feature.

File: dataservice/api/diagnosis/models.py [hunk 2]
```diff
@@ -16,6 +16,9 @@ class Diagnosis(db.Model, Base):
     :param tumor_location: Location of the tumor
     :param age_at_event_days: Age at the time of diagnosis expressed
     in number of days since birth.
+    :param mondo: An ontology that harmonizes diagnosis
```
Comment (znatty22) [inline dataservice/api/diagnosis/models.py]: Are these supposed to be id fields that reference the ontology term (like HPO)? If so, we should probably name them `ncit_id`, `mondo_id`

File: dataservice/api/sample/models.py [hunk 3]
```diff
@@ -18,6 +18,7 @@ class Sample(db.Model, Base):
             acquired, expressed in number of days since birth
     :param tumor_descriptor: The kind of disease present in the tumor
            specimen as related to a specific timepoint
+    :param uberon: Uber-anatomy ontology for anatomical_site
```
Comment (znatty22) [inline dataservice/api/sample/models.py]: Same comment as before. If this is supposed to be the id of the term in Uberon, then it should probably be named `uberon_id`.

Comment (parimalak) [reply dataservice/api/sample/models.py]: 👍

File: dataservice/api/diagnosis/models.py [hunk 4]
```diff
@@ -32,6 +34,10 @@ class Diagnosis(db.Model, Base):
     age_at_event_days = db.Column(db.Integer(),
                                   doc='Age at the time of event occurred in '
                                       'number of days since birth.')
+    mondo_id = db.Column(db.Text(),
+                         doc='An ontology that harmonizes diagnosis')
```
Comment (znatty22) [inline dataservice/api/diagnosis/models.py]: Maybe change the docs to: 
- `The ID of the term from the Monary Disease Ontology which represents a harmonized diagnosis`.
- `The ID of the term from the International Classification of Diseases which represents a harmonized diagnosis.`

File: dataservice/api/phenotype/models.py [hunk 5]
```diff
@@ -19,6 +20,7 @@ class Phenotype(db.Model, Base):
 
     phenotype = db.Column(db.Text())
     hpo_id = db.Column(db.Text())
```
Comment (znatty22) [inline dataservice/api/phenotype/models.py]: Same comment as above with docs

File: dataservice/api/study/models.py [hunk 6]
```diff
@@ -29,6 +30,7 @@ class Study(db.Model, Base):
     version = db.Column(db.Text())
     name = db.Column(db.Text())
     attribution = db.Column(db.Text())
+    release_status = db.Column(db.Text())
```
Comment (znatty22) [inline dataservice/api/study/models.py]: Maybe eventually want to make this an enum

**Review by znatty22 — COMMENTED** [review]
Looks good just minor comments

File: dataservice/api/diagnosis/models.py (hunk 4 above)
Comment (parimalak) [reply dataservice/api/diagnosis/models.py]: 👍

File: dataservice/api/study/models.py (hunk 6 above)
Comment (parimalak) [reply dataservice/api/study/models.py]: yeah, updated in enum issue #48

File: dataservice/api/phenotype/models.py (hunk 5 above)
Comment (parimalak) [reply dataservice/api/phenotype/models.py]: 👍

**Review by znatty22 — APPROVED** [review]
🥇

**Comment by dankolbman** [comment]
I think we're ok to change `original_reference_genome` to `reference_genome`

**Comment by parimalak** [comment]
@dankolbman yep I renamed to reference_genome. you can see the overall changes in PR description

**Review by dankolbman — APPROVED** [review]

**Pull request merged**